src/gcp_scanner/crawl.py

```diff
@@ -67,8 +67,8 @@ def fetch_project_info(project_name: str,
       project_info = response
 
   except Exception:
-    logging.info("Failed to enumerate projects")
-    logging.info(sys.exc_info())
+    logging.error("Failed to enumerate projects")
```

Comment:
I think if we make all our crawler info messages for exceptions to be error messages that might produce a lot of output for our user and be a bit confusing since it is not actually GCP Scanner error but credentials that do not give access to certain GCP Resources. I'd prefer to keep info here.